### ISSUE TITLE
[minor bug] duplicate names

### ISSUE THREAD

**Issue by martinandersen3d** [SjoerdV/ConvertOneNote2MarkDown#15]
when there is duplicate names, one will override the other.

![image](https://user-images.githubusercontent.com/19984581/98201310-aeb5b500-1f2f-11eb-871d-42d784cff6bb.png)

I dont think it needs to be fixed (just an edgecase). Just want to report it.

**Comment by SjoerdV** [comment]
Hi @martinandersen3d are you sure about that? There is a routine that deals with duplicates in the script by appending '_[number]' to the exported markdown:
``` powershell
              # in case multiple pages with the same name exist in a section, postfix the filename
              if ([System.IO.File]::Exists("$($fullexportpathwithoutextension).md")) {
                $pagename = "$($pagename)_$recurrence"
                $recurrence++
              }
```

**Comment by SjoerdV** [comment]
Probably this is now fixed with #23.
